### PR TITLE
[decentraland-estate-size] Add erc721-with-multiplier and decentraland-estate-size strategies

### PR DESCRIPTION
**Pull request by 2fd**
## erc721-with-multiplier 

Allow using an arbitrary multiplier to increase the score

## decentraland-estate-size

Calculate the number of lands that an address own in estates

### CODE REVIEW

File: src/strategies/decentraland-estate-size/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  options,
+  snapshot
+) {
+  const multipler = options.multiplier || 1
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const response = await multicall(
+    network,
+    provider,
+    abi,
+    addresses.map((address: any) => [options.address, 'getLANDsSize', [address]]),
```
Comment (bonustrack) [inline src/strategies/decentraland-estate-size/index.ts:25]: Do you have this data on a subgraph? Somehow this call is taking lot of resources on our node, does this call have complex logic on the smart contract? With this implementation the multicall request may fail if there is too much addresses (~350).

**Review by jaceppo — APPROVED** [review]
guanlong huang

**Review by jaceppo — APPROVED** [review]

File: src/strategies/decentraland-estate-size/index.ts (hunk 1 above)
Comment (2fd) [reply src/strategies/decentraland-estate-size/index.ts]: Yes, it has a complex logic. I'm going to reimplement it using our graph 😄

**Comment by 2fd** [comment]
@bonustrack I just pushed the changes to use the graph, what do you think?

**Review by bonustrack — APPROVED** [review]
It's much faster now, great work! I'll merge the PR it should be live in ~5mn

**Pull request merged**